GAME/DELTAPAK.C

```diff
@@ -501,14 +501,14 @@ void trigger_title_spotcam(int num)//32904(<), 32D9C(<) (F)
 		 * Resetting all outside church sequence animatings to their original position & animation.
 		 */
 		//Man holding umbrella at church area (TITLE.TRC)
-		item = ResetCutanimate(ANIMATING10);
+		item = ResetCutanimate(SPECIAL1_MAN_WALKING);
```

Comment:
I don't understand why there is an additional enum here, objects.h defines all objects.

Reply:
because some cutscenes use weird ids for objects for example in the joby* cutscenes, Sergie uses id 58 (LION_MIP) and in the andy* cutscenes, the demon knight uses id 34 (SWAT_MIP)

Reply:
According to CODE.WAD snippets was it? Core definitely used the OBJECTS.H definitions when calling ResetCutAnimate. 

More-ontopic. I don't know what's going to be achieved here other than confusion. I see you want to ensure that object naming is clear. Be careful though, the same object id in a different level may correspond to a completely different object thus accurate naming will be a tricky task. It's not a good idea to have two enums with exactly the same information.